hack/crd-schema-checker.sh

```diff
@@ -28,5 +28,7 @@ for crd in config/crd/bases/*.yaml; do
     $CHECKER check-manifests \
         $CHECKER_ARGS \
         --existing-crd-filename="$TMP_DIR/$crd" \
-        --new-crd-filename="$crd"
+        --new-crd-filename="$crd" \
+        --disabled-validators NoMaps \
+        --disabled-validators NoNewRequiredFields
```

Comment:
we should not disable it. if there is a new required field and its ok, we should instead override the failing CI job. otherwise it would need a next PR to enable it again.

Reply:
Ok np! I reverted this bit and validated the links from the controller itself.